lib/elixir_avro/generator/content.ex

```diff
@@ -10,35 +10,48 @@ defmodule ElixirAvro.Generator.Content do
         ) :: map
   def modules_content_from_schema(root_schema_content, read_schema_fun, module_prefix) do
     root_schema_content
-    |> ElixirAvro.SchemaParser.parse(read_schema_fun)
-    |> Enum.map(fn {_fullname, erlavro_type} -> module_content(erlavro_type, module_prefix) end)
+    |> ElixirAvro.Schema.Parser.parse(read_schema_fun)
+    |> Enum.map(fn {_fullname, schema_type} -> module_content(schema_type, module_prefix) end)
     |> Enum.into(%{})
   end
 
-  @spec module_content(erlavro_schema_parsed :: tuple, module_prefix :: String.t()) ::
-          {String.t(), String.t()}
-  defp module_content(
-         erlavro_schema_parsed,
-         module_prefix
-       ) do
-    moduledoc = module_doc(erlavro_schema_parsed)
-    module_name = module_name(erlavro_schema_parsed, module_prefix)
+  @spec module_content(
+          schema :: :avro.avro_type(),
+          module_prefix :: String.t()
+        ) :: {String.t(), String.t()}
+  defp module_content(schema, module_prefix) do
+    moduledoc = module_doc(schema)
+    module_name = module_name(schema, module_prefix)
+    specific_bindings = get_spefic_bindings(schema)
```

Comment:
```suggestion
    specific_bindings = get_specific_bindings(schema)
```

I think there's a typo